frontend/assets/stylesheets/components/_related.scss

```diff
@@ -0,0 +1,51 @@
+/* ==========================================================================
+   Related Content
+   ========================================================================== */
+
+.related-entry {
+    display: table;
+    width: 100%;
+    background-color: $c-neutral8;
+    border-top: 1px solid $c-border-neutral;
+    padding: rem($gs-gutter / 4);
+    margin-bottom: rem($gs-baseline * 3);
+
+    @include mq(tablet) {
+        display: block;
+        border: 0 none;
```

Comment:
@wpf500 You are right, for some reason I got into the habit of writing `border: 0 none;`